ui/app/router.js

```diff
@@ -108,6 +108,44 @@ Router.map(function () {
           this.route('index', { path: '/' });
           this.route('create');
         });
+        this.route('oidc', function () {
```

Comment:
I see the need for the individual routes, and this is my preferred approach, but are there any concerns with having a very large router file? If we follow this pattern in the future (mind you I don't think we'll add many more secret engines, but we could...).

Reply:
I don't think there are any concerns with how big the router gets or at least I haven't heard of any issues with performance. I think the benefits of having clearly defined routes outweigh having a single route and then dynamically loading different resource types.